tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/PoPTests.NetFwk.cs

```diff
@@ -58,7 +58,7 @@ public void TestInitialize()
 
             if (_keyVault == null)
             {
-                _keyVault = new KeyVaultSecretsProvider(KeyVaultInstance.MsalTeam);
+                _keyVault = new KeyVaultSecretsProvider(KeyVaultInstance.MSIDLab);
```

Comment:
There is a lab API query to get all these details, we should at some point shift to using the query `https://msidlab.com/api/config?publicClient=no&signInAudience=azureadmyorg`